### ISSUE TITLE
Incorrect loss calculation 

### ISSUE THREAD

**Issue by wenc13** [princeton-vl/SimpleView#13]
@imankgoyal 
The classification loss at line 123 of the file "main.py" is incorrect,

> loss = F.cross_entropy(logit, label)

because at line 147 of file "pointnet_pyt/pointnet/model.py", the output of the fully-connected layer has been applied the F.log_softmax().

> return F.log_softmax(x, dim=1), trans, trans_feat

**Comment by imankgoyal** [comment]
Hi @wenc13 ,

I don't think the loss calculation is incorrect here. 

[F.cross_entropy](https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html) can take unnormalized logits as input. The output should not change if the logits are normalized.  `F.log_softmax(x, dim=1)` simply normalizes the logits. 

```
Python 3.8.17 (default, Jul  5 2023, 21:04:15) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.12.2 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import torch

In [2]:  import torch.nn.functional as F

In [3]: input = torch.randn(3, 5, requires_grad=True)

In [4]:  target = torch.randint(5, (3,), dtype=torch.int64)

In [5]: F.cross_entropy(input, target)
Out[5]: tensor(2.5989, grad_fn=<NllLossBackward0>)

In [6]: F.cross_entropy(F.log_softmax(input), target)
<ipython-input-6-83a7db40dfc5>:1: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  F.cross_entropy(F.log_softmax(input), target)
Out[6]: tensor(2.5989, grad_fn=<NllLossBackward0>)
```

**Comment by imankgoyal** [comment]
Closing because of inactivity. Please feel free to reopen.
